src/Forms/Shared/Samples/Map/DisplayOverviewMap/readme.metadata.json

```diff
@@ -0,0 +1,41 @@
+{
+    "category": "Map",
+    "description": "Include an overview or inset map as an additional map view to show the wider context of the primary view. ",
+    "formal_name": "DisplayOverviewMap",
+    "ignore": false,
+    "images": [
+        "screenshot.png"
```

Comment:
Screenshot needs to be updated to match actual screenshot file for the Developers site.